packages/util/src/withdrawal.ts

```diff
@@ -103,6 +103,15 @@ export class Withdrawal {
     return Withdrawal.toBufferArray(this)
   }
 
+  toValue() {
```

Comment:
This is nice!  I wonder if we should add something like this for blob txns?

Reply:
yes its good :+1: picked idea from ssz code structure